homepage.css

```diff
@@ -364,3 +363,10 @@ ul li:hover ul{
 	transform: translate(-50%, -50%);
 	text-align: center;
 }
+
+.text a{
```

Comment:
I add a overlay techniques so I set up the color font, fonts , color size so the text when I hover on. It also be the link to like to other query